docs/user/debugging.md

```diff
@@ -4,154 +4,243 @@
-    logged into the Stackdriver Console of the same project that Turbinia is in.
+*   **API, Server and Worker logs**: By default these logs are in
+    `LOG_DIR/<gke-pod-name>.<gke-node-name>.log` for GKE installations and
+    `LOG_DIR/<hostname>.log` for local installations.  For GKE the pod names
+    will contain `api`, `server` or `worker` depending on their role in the
+    cluster.  Note that these paths may be overridden by the `TURBINIA_LOG_FILE`
+    environment variable, so depending on your installation configuration it may
+    be at a different location.
+*   **Task logs**: These are logs that are generated by Tasks running in the
+    Worker. These logs are in the Task output directory, which is in
+    `OUTPUT_DIR/<request_id>/<epoch>-<task_id>-<task_name>/`.  The task logs
+    will also show up as saved output for the task and the paths can be
+    determined by doing `turbinia-client status task <task-id> -a` (note that -a
+    is required to show all data associated with the tasks).  To retrieve
+    all files associated with a given task, including the task logs, you can
+    run `turbinia-client result task <task id>` and it will download a
+    compressed file with all of the output.
+*   **Task dependency execution logs**: These are the logs that are generated by
+    binaries executed from a Task. For example, Plaso generates its own log
+    file, and that gets saved by the Task. As with the Task logs, these are
+    stored in the Task output directory.  See the above `Task logs` section for
+    how to determine the paths and retrieve the output.
+*   **Google Cloud Error Reporting / Stackdriver Logs**: If you have deployed
+    Turbinia in GKE and/or have `STACKDRIVER_TRACEBACK` enabled in the config,
+    the logs and/or traceback exceptions will also be logged to GCP Monitoring
+    for the same project that Turbinia is in.
 
 ## Request Debugging
 
 ### Finding previous request data
-By default `turbiniactl` will make a processing request and immediately return.
-It will print out the request ID when it makes the request, and you can use this to
-see the current status of the request with `turbiniactl status -r <request id>`.  If
-you specify the `-w` flag (`turbiniactl -w status -r <request id>`), the client will
-wait for all Tasks to complete and then return.  Killing the client will not affect
-the processing, and you can resume waiting for the task results with the same command.
-
-If you do not have your request ID, you can list all recent requests like this:
-```
-$ turbiniactl status -i
+When processing requests are submitted via the client (e.g.
+`turbinia-client submit <evidence type>`) it will print out the request ID for
+that request. This can be used with
+`turbinia-client status request <request id>` to get the current state of the
+processing request.  Alternately you can also use the [web
+ui](turbinia-web-ui.md) to get details as well.  For more details on using the
```

Comment:
Change to *Alternatively 

And  maybe change to "to get details and download output as well."

Reply:
Thanks!  Both Done.   TIL I've been using this word wrong my entire life :)